pkg/controller/servicemesh/memberroll/ovs_networkpolicy.go

```diff
@@ -134,7 +134,7 @@ func (s *networkPolicyStrategy) removeNamespaceFromMesh(ctx context.Context, nam
 		for _, np := range npList.Items {
 			logger.Info("deleting NetworkPolicy for mesh", "NetworkPolicy", np.GetName())
 			err = s.Client.Delete(ctx, &np)
-			if err != nil {
+			if err != nil && !(errors.IsNotFound(err) || errors.IsGone(err)) {
```

Comment:
Same as above, do we need to reset `err` to `nil`?

Reply:
Nope. It isn't used anywhere else.